Example/ContentView.swift

```diff
@@ -43,6 +43,7 @@ struct ContentView: View {
     @State var me: User = User() // currently logged in user
     @State var lastRegisteredId: Int? // last registered user, this is also who you can delete
     @State var isEbike: Bool = false
+	@State var termsList: [TermsAndConditions] = []
```

Comment:
I think it is just this file that is wrong and I am using the project settings that match all the other files.

Reply:
Now that I'm going back over it, it's honestly hard to tell—it seems like there might be a mish mash of different indentation settings used. This one can slide, but it's probably worth doing a pass of the project as a whole to standardise something: it makes the code difficult to read when the indentation is all squiggly.

Reply:
it was this: https://github.com/Zone5-Cloud/z5-sdk-swift/commit/d8c7a484bdbc66948d7cb9d6aa2b9b2ee71f01eb
I was going to fix it after the commit and never got around to it. I'll do it now. It will make my changes in this ContentView file hard to see. But as long as you've already reviewed it...

Reply:
done